### PR TITLE
Try running in Travis with ghc-8.8.1

### PR DESCRIPTION
**Pull request by neurogoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -92,3 +92,14 @@ matrix:
...
         - libgmp-dev
+  - env: BUILD=cabal GHCVER=8.8.1 # PEDANTIC=YES
+    compiler: ghc-8.8.1
+    addons:
+      apt:
+        sources:
+        - hvr-ghc
+        packages:
+        - libfftw3-dev
+        - cabal-install-head
```
Comment (phadej) [inline .travis.yml]: use `-3.0`

Comment (phadej) [reply .travis.yml]: ... and I'd suggest starting with `futurice-haskell-prelude`, that's the first thing which would need help.

Comment (neurogoo) [reply .travis.yml]: Heh true, forgot cabal update already :)

**Comment by neurogoo** [comment]
Still need to either vendor/wait/replace few packages before continuing work:
- Haxl
- algebraic-graphs (there is already patch in Github)
- pretty-compact
- machines-* (low activity on those in recent years so may need to be replaced)

**Comment by neurogoo** [comment]
Merged needed changes needed to support ghc-8.8 from another branch

**Pull request closed without merging**